packages/web-app-activities/src/components/ActivityItem.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div class="oc-flex oc-flex-middle activity-item">
+    <div class="oc-flex oc-flex-middle">
+      <oc-avatar :width="36" :user-name="activity.template.variables.user.displayName" />
+      <span class="oc-ml-s" v-text="activity.template.variables.user.displayName" />
+    </div>
+    <div>activity unknown</div>
```

Comment:
do we still want this translated?

Reply:
i dont think so, this is just a placeholder, an no one should ever see this in production